eve/workers/end2end/scripts/build-load-operator.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -exu
+
+OPERATOR_IMAGE=${1:-zenko-operator:latest}
+OPERATOR_REPO=${2:-git@github.com:scality/zenko-operator.git}
+OPERATOR_BRANCH=${3:-development/1.0}
+
+git clone ${OPERATOR_REPO} -b ${OPERATOR_BRANCH} operator
```

Comment:
Because we are cloning the zenko operator directly there may be a time where a zenko operator update breaks zenko but we wouldn't know until someone updated zenko. I wonder if we could run a zenko test after each operator update or something similar. Or maybe this is already handled in some way.

Reply:
I believe this should be done by scheduled job like the current nightly.

Reply:
+1 here, same story goes for https://github.com/scality/Zenko/pull/1054/files#diff-73e74177595a435f88a0701e605cf08eR46

I believe we can start like that, but focus on fixing a version ASAP.